src/dataflow/src/render/reduce.rs

```diff
@@ -1202,8 +1344,7 @@ where
 
     collection
         .arrange_by_self()
-        .reduce_abelian::<_, OrdValSpine<_, _, _, _>>(
-            "ReduceAccumulable", {
+        .reduce_abelian::<_, OrdValSpine<_, _, _, _>>("ReduceAccumulable", {
```

Comment:
This is .. what like `rustfmt` changed or something?

Reply:
There was a string literal that was longer than 100 chars that was causing rustfmt to just give up. I split that up over two lines so now rustfmt is reformatting everything. 🤷🏽‍♂️